src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartTokenFormatterFormatRangeTests.cs

```diff
@@ -444,14 +444,40 @@ class Class
 
             var expected = @"using System;
 class Class
+{
+    int Prop {
+        get { return 1;
+        }";
+
+            await AutoFormatOnCloseBraceAsync(code, expected, SyntaxKind.OpenBraceToken);
+        }
+
+        [WpfFact]
+        [WorkItem(16984, "https://github.com/dotnet/roslyn/issues/16984")]
+        [Trait(Traits.Feature, Traits.Features.SmartTokenFormatting)]
+        public async Task AccessorList5b()
+        {
+            var code = @"using System;
+class Class
+{
+    int Prop {
+        get { return 1;   
+}$$
+}
+}";
+
+            var expected = @"using System;
+class Class
 {
     int Prop {
         get
         {
             return 1;
-        }";
+        }
+}
+}";
 
-            await AutoFormatOnCloseBraceAsync(code, expected, SyntaxKind.GetKeyword);
+            await AutoFormatOnCloseBraceAsync(code, expected, SyntaxKind.OpenBraceToken);
```

Comment:
@heejaechang What is this argument for?

Reply:
token tells you what token range start with. this could be use the marker stuff to mark what start token it is looking for rather than give in token kind.